src/main/java/blob/parser/DateTimeParser.java

```diff
@@ -0,0 +1,58 @@
+package blob.parser;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeParseException;
+
+import java.util.Locale;
+
+public class DateTimeParser {
```

Comment:
Creating a DateTimeParser class really helps out a lot in parsing the different types of date formats available. Great job!